### DISCUSSION TITLE
Creation automatic adif file

### DISCUSSION THREAD

**Discussion by f4emm** [foldynl/QLog#136] [Ideas]
Hello,
Would it be possible to have the creation of an automatic adif file?
Thanks.
Have a good day.
F4EMM

**Comment by foldynl** [comment]
What exactly does that mean? when should the adif file be created?

ADX (not ADI) is automatically created as a backup every time QLog is started

**Reply by f4emm** [reply]
Hello,
Ok, and where is this file located?
Thanks.
Have a nice week end.

**Comment by foldynl** [comment]
look at the wiki. 

https://github.com/foldynl/QLog/wiki/Autobackup

keep in mind, it is ADX (not ADI). because ADI is not prepared to contain an international characters. Therefore I chose ADX format that allows to save also the international characters.
